bindings/java/java/src/main/java/org/ethereum/evmc/EvmcVm.java

```diff
@@ -0,0 +1,113 @@
+package org.ethereum.evmc;
+
+import java.nio.ByteBuffer;
+import java.util.ArrayList;
+import java.util.Collections;
+import java.util.List;
+
+/**
+ * The Java interface to the evm instance.
+ *
+ * <p>Defines the Java methods capable of accessing the evm implementation.
+ */
+public final class EvmcVm {
+
+  static {
+    try {
+      System.load(System.getProperty("user.dir") + "/../c/build/evmc.so");
+    } catch (UnsatisfiedLinkError e) {
+      System.err.println("Native code library failed to load.\n" + e);
+      System.exit(1);
+    }
+  }
+
+  public static native int test(ByteBuffer buff);
```

Comment:
if this is just for testing, can we make it package visible only?

Reply:
deleted